src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -513,6 +518,87 @@ void RegisterNotification(DocumentChange notification)
             }
         }
 
+        private long GetStartEtagByStartChangeVector(ChangeVectorEntry[] startChangeVector, DocumentsOperationContext dbContext, SubscriptionCriteria criteria)
```

Comment:
Ignoring this method because of the expected changes discussed in length elsewhere